workorders.lic

```diff
@@ -84,7 +84,12 @@ class WorkOrders
       if repair
         repair_items(info, tools)
       else
-        sew_items(info, item, quantity)
+	recipe = @recipes.find { |r| r['name'] == item }
```

Comment:
I was planning that for a later refactor, work orders as a whole needs some clean up and standardization.